components/scaffold/table/helpers.go

```diff
@@ -106,6 +106,15 @@ func WithSortURL(sortURL string) ColumnOpt {
 	}
 }
 
+// WithNotSortable disables sorting for a column
+func WithNotSortable() ColumnOpt {
```

Comment:
This is a little weird, why not make column not sortable by default and add Sortable as needed?